subit.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+
+
+
+    <title>Project 1</title>
+    <link type="text/css" rel="stylesheet" href="style.css">
+    
+</head>
+
+<body>
+    <forms>
+        <h1><b>My GuestBook</b></h1>
+        <p>Please sign in my Guest book.Thanks</p>
+        <div class="Name">
+            <label for="name" id="name"><b>What is your name</b></label>
+            <input  class="me" type="text">
+        </div>
+        <div class="email">
+            <label for="email" id="email"><b>What is your eamil address?</b></label>
+            <div class="ru"><input type="email"></div>
+        </div>
+        <div class="check">
+            <div id="check"><b>Please check all the things you like</b></div>
+            <div class="duir">
+                <p><input id='1' type="checkbox">I really like your website</p>
+                <p><input id='1' type="checkbox">One of the best website i have seen</p>
+                <p><input id='1' type="checkbox">I wish my site looked so beautiful</p>
+                <p><input id='1' type="checkbox">I have no taste.I am pretty dense so your site doesn't do much for</p>
+            </div>
+        </div>
+        <div class="'choose">
+            <div id="choose"><b>Choose the thing you like the most in my website</b></div>
+            <div class="dido">
+                <p><input id='2' type="radio" name="a">All the gorgeous picture</p>
+                <p><input id='2' type="radio" name="a">All the gorgeous picture of ur cat</p>
+                <p><input id='2' type="radio" name="a">The inspiring recap of our amazing suburban childhood</p>
+                <p><input id='2' type="radio" nmae="a">The detailled list of all of our ellvis memorabillia's</p>
```

Comment:
`id` has to be unique

Reply:
Ok,Should i resubmit it for you to approve it or will you approve it.if should re submit how?

Reply:
You push push your changes if you want. Refer to [this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork), for how to push changes.